examples/04-existing-vpc.yaml

```diff
@@ -8,19 +8,25 @@ metadata:
   region: eu-north-1
 
 vpc:
+  id: "vpc-0dd338ecf29863c55"  # (optional, must match VPC ID used for each subnet below)
+  cidr: "192.168.0.0/16"       # (optional, must match CIDR used by the given VPC)
   subnets:
-    private:
-      # NOTE: you cannot run this example directly,
-      # you will need to set actual subenet IDs here
-      eu-north-1a:
-        id: subnet-a123456
-      eu-north-1b:
-        id: subnet-b123456
-      eu-north-1c:
-        id: subnet-c123456
+    # must provide 'Private' and/or 'Public' subnets by availibility zone as shown
+    Private:
```

Comment:
We should make this lowercase, but that would a bigger change and we'd have to bump the API version.